kythe/go/extractors/config/extractor.go

```diff
@@ -43,11 +43,11 @@ type Repo struct {
 	ConfigPath string
 }
 
-// GitCopier creates a Repo.Copier that fetches repos via git commandline.
-func GitCopier(ctx context.Context, repoURI string) func(string) error {
-	return func(targetDir string) error {
+// GitCopier creates a Repo.Clone that fetches repos via git commandline.
```

Comment:
Since we didn't define a type for this, I suggest writing this out as prose rather than like a type name, e.g.,

    // GitCloner returns a function that clones a repository via the git command line.

perhaps.

Reply:
oh yea good catch, actually I had this as a type and then removed it, but when I fixed the comment I left it as though it was a type.